.hermes/conveyor/work-3d8d9b32/adr.md

```
@@ -0,0 +1,60 @@
+# ADR-0013: Close Issue #289 — Severity::Info Already Maps to "info"
+
+## Status
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)